torch/csrc/jit/serialization/python_print.cpp

```diff
@@ -739,6 +739,27 @@ struct PythonPrintImpl {
         ss << "fork(" << name << ")";
         printOutputDefinition(node, ss.str());
       } break;
+      case prim::Enter: {
+        const auto in = node->inputs().at(0);
+        const auto out = node->outputs().at(0);
+        indent();
+        body_ << "with " << useOf(in);
+        if (out->uses().size() > 0) {
+          assignValue(out, genUniqueNameFor(out));
+          body_ << " as " << useOf(out);
+        }
+        body_ << ":\n";
+        level++;
+      } break;
+      case prim::Exit: {
+        // If the previous node is a prim::Enter, the with block the generated
+        // this Enter/Exit pair must have been empty.
+        if (node->prev()->kind() == prim::Enter) {
+          indent();
+          body_ << "pass\n";
+        }
+        level--;
+      } break;
       case prim::Function: {
```

Comment:
Can you add `as` and `with` to the list of `reserved_names`? i don't think this is BC-breaking bc in python right now if you try to assign a variable to either you'll get a hard error
```
>>> as = 4
  File "<stdin>", line 1
    as = 4
SyntaxError: invalid syntax
>>> with = 5
  File "<stdin>", line 1
    with = 5
         ^
SyntaxError: invalid syntax
```